@model System.Web.Mvc.HandleErrorInfo
@{
    ViewBag.Title = "Application Exception";
}


<div class="card col-md-12 p-0 mt-4">
    <div class="card-header bg-gradient-danger p-3 text-left text-white">
        <h4 class="m-0">Exception Alert <i class="fa fa-frown-o align-middle " aria-hidden="true"></i></h4>
    </div>
    <div class="card-body">
        <table class="table table-bordered" style="font-size:1rem">
            <tr>
                <td><strong>Entity</strong></td>
                <td>@Model.ControllerName</td>
            </tr>

            <tr>
                <td><strong>Action</strong></td>
                <td>@Model.ActionName</td>
            </tr>

            <tr>
                <td><strong>Exception</strong></td>
                <td>
                    @(Model.Exception.Message == "Deleted" ? "Record already deleted by another user !" : Model.Exception.GetType().Name+" : "+Model.Exception.Message)
                    @if (Model.Exception is System.Data.Entity.Infrastructure.DbUpdateConcurrencyException)
                    {
                        if (Model.ActionName == "Edit")
                        {
                            <label>
                                The record you attempted to modify was modified/deleted by another user after you got the original values.Click below button to get updated record.
                            </label>
                        }
                        if (Model.ActionName == "Delete")
                        {
                            <label>
                                The record you attempted to delete was modified/deleted by another user after you got the original values. Click below button to get updated record.
                            </label>
                        }

                    }

                </td>
            </tr>


        </table>

        <div class="text-right">
            <a class="btn btn-primary btn-sm" href="@Request.Url.PathAndQuery">Refresh</a>
            <a class="btn btn-primary btn-sm" href="@Request.UrlReferrer.PathAndQuery">Go Back</a>
        </div>
    </div>
</div>
@*<div class="row">
        <h1 class="page-title">
            <i class="glyphicon glyphicon-exclamation-sign text-primary"></i> Exception Alert
        </h1>
        <p class="text-primary">
            Entity :<b> @Model.ControllerName </b><br>
            Action:<b> @Model.ActionName </b><br>
            Exception:  @(Model.Exception.Message == "Deleted" ? "Record already deleted by another user !" : Model.Exception.GetType().Name+" : "+Model.Exception.Message)
        </p>
        <hr />
        <p class="text-primary">
            @if (Model.Exception is System.Data.Entity.Infrastructure.DbUpdateConcurrencyException)
            {
                if (Model.ActionName == "Edit")
                {
                    <label class="text-primary">
                        The record you attempted to modify was modified/deleted by another user after you got the original values.Click below button to get updated record.
                    </label>
                }
                if (Model.ActionName == "Delete")
                {
                    <label>
                        The record you attempted to delete was modified/deleted by another user after you got the original values. Click below button to get updated record.
                    </label>
                }
                <a class="btn btn-default btn-sm" href="@Request.Url.PathAndQuery">Refresh</a>
            }
            else
            {
                <a class="btn btn-default btn-sm" href="@Request.UrlReferrer.PathAndQuery">Go Back</a>
            }
        </p>
    </div>*@
